addons/l10n_nl/views/res_partner_views.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="l10n_nl_view_partner_form" model="ir.ui.view">
```

Comment:
`id` should be `view_partner_form_inherit_l10n_nl` with `res.partner.form.inherit.l10n_nl` as name